File: _study/COS_PRO_2/python/stage3/10.py
# -*- coding: utf-8 -*-
# UTF-8 encoding when using korean

def solution(arr):
    answer = 0
    for i in arr:
        if i/2 in arr:
            answer += 1
    return answer


arr = [4, 8, 3, 6, 7]
ret = solution(arr)

#[실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
print("solution 함수의 반환 값은", ret, "입니다.")

File: _study/JejuCodingBasecamp/codefestival_js/Part01/10.js
const ps = require('../modules/ps');

const solution = (n) => {
  let answer = '';

  for (let i = 0; i < n; i++) {
    for (let j = n-i-1; j > 0; j--) {
      answer += ' ';
    }
    for (let j = 0; j < (i*2)+1; j++) {
      answer += '*';
    }
    answer += '\n';
  }

  return answer;
};

ps.main(async () => {
  console.log('입력');
  let n = parseInt(await ps.readLine()); // 문자열을 입력 받아 정수 반환

  console.log(solution(n));
});
